src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -923,15 +923,29 @@ spring.datasource.username=notification
 spring.datasource.password=notification
 ----
 
-===== Prepare the database purge operation
+===== Database purge operation
 Additionally, a `vacuum` operation does not by default remove orphaned Large Objects from the database.
 Identification and removal of these objects is performed by the `vacuumlo` command.
 
 As the ProActive database contains many large objects, it is essential to apply both `vacuumlo` and `vacuum` operations in order to control the disk space usage over time.
 
-Prior to execute the above operations, a preliminary configuration step is necessary (this preliminary step is done once for a given ProActive installation).
+In addition, prior to execute the purge operation, a preliminary configuration step is necessary (this preliminary step is done once for a given ProActive installation).
 
-After the first start of the ProActive server, the following SQL queries will alter several tables of the database (this is needed due to a PotsgreSQL driver issue):
+In the following, we explain, step by step, the different steps to follow in order to perform the database purge operation:
```

Comment:
In the following section, we describe the step by step procedure to perform the database purge operations.